setup.cfg

```diff
@@ -82,6 +82,30 @@ aiosqlite =
 sqlcipher =
     sqlcipher3_binary
 
+# legacy pre-pep-685 names
+mariadb_connector =
```

Comment:
so, towards us being able to move to a pyproject.toml file easily, the request here is to not use the aliases for these legacy names and just point them to their ultimate dependencies directly (yes, duplicating what we have up above).

the goal here is that this batch of names is moved out to the margins where it can be removed at some point (maybe even 2.1, but maybe 2.2, etc) all at once.

Reply:
so from my point of view we can delay all the "remove the alias" to the PR that migrates to pyproject. As far as I know the toml version does not support aliases, but maybe there is a way that I'm not aware of